lib/tests/misc.nix

```diff
@@ -969,6 +969,13 @@ runTests {
 
   testToSentenceCasePath = testingThrow (strings.toSentenceCase ./.);
 
+  testToCamelCase = {
+    expr = strings.toCamelCase "hello world";
+    expected = "helloWorld";
+  };
```

Comment:
If this function has the ability to 
>    Handles snake_case, PascalCase,
>    kebab-case strings as well as strings delimited by spaces

These features should have tests as well.

Reply:
Added :)